x/wasm/keeper/gas_register.go

```diff
@@ -27,12 +27,12 @@ const (
 	// such as hardcoding them in contracts.
 	//
 	// Please note that all gas prices returned to wasmvm should have this multiplied.
-	DefaultGasMultiplier uint64 = 15_000_000
+	DefaultGasMultiplier uint64 = 140_000_000
```

Comment:
Good changes.

I would add some text where this came from.
Minimally linking to the Benchmarking PR and the discussion.
(For these 3 items)